cdn-server/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@wundergraph/cdn-server",
```

Comment:
Same here. What makes a CDN in that context? A CDN is optimized for delivery. I suggest to give a more descriptive name.

Reply:
We stick with the name.